include/geos/operation/polygonize/PolygonizeGraph.h

```diff
@@ -175,15 +175,15 @@ class GEOS_DLL PolygonizeGraph: public planargraph::PlanarGraph {
 	 * The labelling allows detecting cut edges.
 	 *
 	 * @param dirEdgesIn  a list of the DirectedEdges in the graph
-	 * @param dirEdgesOut each ring found will be pushed here
+	 * @result vector that contains each ring found
 	 */
-	static void findLabeledEdgeRings(
-			std::vector<planargraph::DirectedEdge*> &dirEdgesIn,
-			std::vector<PolygonizeDirectedEdge*> &dirEdgesOut);
+	std::vector<PolygonizeDirectedEdge*>
+	findLabeledEdgeRings(
+			std::vector<planargraph::DirectedEdge*> &dirEdgesIn);
```

Comment:
The output parameter was used to avoid a vector copy, passing a reference to an existing vector allows appending to a pre-allocated vector (for example)

Reply:
it is using the move constructor,
you actually want that on the parameter if you are going to "continue filling" the vector